.semaphore/semaphore.yml

```diff
@@ -34,8 +39,8 @@ blocks:
       jobs:
         - name: Compile with Jekyll
           commands:
-            - bundle exec jekyll build --baseurl "/$SEMAPHORE_GIT_BRANCH/$SEMAPHORE_GIT_SHA"
-            - cache store site-$SEMAPHORE_GIT_SHA _site
+            - bundle exec jekyll build --baseurl "/$SEMAPHORE_GIT_BRANCH"
+            - cache store site-$SEMAPHORE_GIT_BRANCH _site
```

Comment:
needed to change this since I removed the `$SEMAPHORE_GIT_SHA`

Reply:
The SHA was removed from the deployment path, but you might want to use it for the cache key as before. Consider a scenario where you went back to an older job for a branch and hit the deploy to staging button. It would not deploy the site from the SHA that this older job built; semaphore would redeploy the latest build